osu.Game.Rulesets.Osu/Objects/SliderTick.cs

```diff
@@ -1,10 +1,22 @@
 ﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
 // Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
 
+using osu.Game.Beatmaps;
+using osu.Game.Beatmaps.ControlPoints;
+
 namespace osu.Game.Rulesets.Osu.Objects
 {
     public class SliderTick : OsuHitObject
     {
         public int SpanIndex { get; set; }
+        public double SliderStartTime { get; set; }
+
+        protected override void ApplyDefaultsToSelf(ControlPointInfo controlPointInfo, BeatmapDifficulty difficulty)
+        {
+            base.ApplyDefaultsToSelf(controlPointInfo, difficulty);
+
+            // SliderTicks appear earlier and earlier going further into a Slider.
+            TimePreempt = StartTime - ((StartTime - SliderStartTime) / 2 + SliderStartTime - TimeFadein * 0.66f);

```

Comment:
Shouldn't this really be `SpanStartTime` or something similar? Kinda looks a bit weird when ticks are overlapping during a repeat. This seems to be what osu-stable does.

Reply:
This is what I got from stable code and it seems to behave identical now. I'll look into trying SpanStartTime.